news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+## A new front-office theme is being built 
+
+For a long time, many community members pushed for a rework of the Classic front-office theme. The maintainer team is happy to announce that a new front-office theme is being built, and you can find [here](https://github.com/PrestaShop/theme-refacto).
+
+## Why such a decision?
+
+The PrestaShop community provided a [lot of feedback](https://github.com/PrestaShop/PrestaShop/issues/14533) and ideas, with suggestions like:
+
+-   Upgrade to Bootstrap 5
+-   Rework the structure inside  `src`  directory, which was  `_dev`  before.
+-   Remove the `core.js` file. Theme developers can't modify anything inside it or even remove it from their theme.
+-   Use [lazy loading](https://en.wikipedia.org/wiki/Lazy_loading).
+-   Choose, and use some CSS class naming strategy consistently, like [BEM](https://github.com/getbem/getbem.com/). This is more maintainable and easier to read.
+-   Use [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD) instead of microdata attributes.
+-   Add as many hooks as possible to extend the base theme.
+-   Separate the PrestaShop styling from the Bootstrap raw styling.
+-   Reduce the number of dependencies.
+-   Stop using jQuery in the theme. This would be the first step because modules use it anyway, so we can't remove it.
+-   Most of the time, JavaScript completely replaces some big parts of the content. Instead, it should update only a small part of it when needed.
+-   Add as many events as possible in the JavaScript part so modules and themes can use them.
+-   Use as many Bootstrap components as possible.
+
+It was pretty impossible to fix all these issues on the classic theme without breaking the whole ecosystem of modules and community themes. We decided that it was time for a change and a new theme. We can't change everything, but we can significantly improve the default PrestaShop theme.
+
+## I would like to try it
+
+Clone it from the [theme repository](https://github.com/PrestaShop/theme-refacto), run `npm install`, `npm run build`, activate the theme and off you go!
+
+## Good base for theme developers
+
+In the new theme, we separated the CSS to two packs. First one is the functionality part, the second is purely for styling.
+
+Commenting out `@import "custom/_custom";` in theme.scss file will get you basically a pure Boostrap 5 theme base. You can get working immediately and not waste time by cleaning things you don't need.
+
+## When will it be released?
+
+Because of the complexity of this task, we decided to avoid writing about the release date. This means that depending on the time every people involved can work on it, the first version of the theme can be released in PrestaShop v8, v9, or later.
```

Comment:
```suggestion
Because of the complexity of this task, we decided to avoid writing about the release date. This means that depending on the time involved people can spend on it, the first version of the theme could be released in either PrestaShop v8, v9, or later.
```